include/ajax.tasks.php

```diff
@@ -424,7 +422,7 @@ function massProcess($action, $w=null)  {
 
             // Assume success
             if ($i==$count) {
-                $msg = sprintf(__('Successfully %s %s.'),
+                $msg = sprintf(__('Successfully %1$s %2$s.' /* Tokens are <actioned> <x selected task(s)> */),
```

Comment:
I'm so sorry this got lost in the many notifications...Yes every thing should stay the same 👍